<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="0" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-17T12:22:28 IST" name="Default suite" finished-at="2021-12-17T12:22:28 IST" duration-ms="99">
    <groups>
      <group name="Regression Testing">
        <method signature="TC001_CreateOrganizationTest.CreatOrganization()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" name="CreatOrganization" class="com.login.TC001_CreateOrganizationTest"/>
      </group> <!-- Regression Testing -->
    </groups>
    <test started-at="2021-12-17T12:22:28 IST" name="Default test" finished-at="2021-12-17T12:22:28 IST" duration-ms="99">
      <class name="com.login.TC001_CreateOrganizationTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" started-at="2021-12-17T12:22:28 IST" name="setUp" finished-at="2021-12-17T12:22:28 IST" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="BrowserLunchandGetUrl()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" started-at="2021-12-17T12:22:28 IST" name="BrowserLunchandGetUrl" finished-at="2021-12-17T12:22:28 IST" duration-ms="0" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.Vitiger.POMclasses.BaseClass.BrowserLunchandGetUrl(BaseClass.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BrowserLunchandGetUrl -->
        <test-method is-config="true" signature="LoginintoApp()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" started-at="2021-12-17T12:22:28 IST" name="LoginintoApp" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1639723948483" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginintoApp -->
        <test-method signature="CreatOrganization()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" started-at="2021-12-17T12:22:28 IST" name="CreatOrganization" finished-at="2021-12-17T12:22:28 IST" duration-ms="1" status="SKIP">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.Vitiger.POMclasses.BaseClass.BrowserLunchandGetUrl(BaseClass.java:40)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CreatOrganization -->
        <test-method is-config="true" signature="LogoutApp()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" started-at="2021-12-17T12:22:28 IST" name="LogoutApp" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1639723948496" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogoutApp -->
        <test-method is-config="true" signature="CloseBrowser()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" started-at="2021-12-17T12:22:28 IST" name="CloseBrowser" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1639723948502" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseBrowser -->
        <test-method is-config="true" signature="disconnectfromDB()[pri:0, instance:com.login.TC001_CreateOrganizationTest@6404f418]" started-at="2021-12-17T12:22:28 IST" name="disconnectfromDB" finished-at="2021-12-17T12:22:28 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- disconnectfromDB -->
      </class> <!-- com.login.TC001_CreateOrganizationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
